_partners/appbaseio.md

```diff
@@ -0,0 +1,5 @@
+---
+name: Appbase.io
+logo: '/assets/media/partners/appbase-io.png'
```

Comment:
Do you have a transparent background png? It will look cleaner and more consistent.